packages/create-ripple-app/saofile.js

```diff
@@ -237,6 +250,18 @@ module.exports = {
       }
     ]
 
+    const removePath = (path, actions) => {
+      if (fs.existsSync(`${this.outDir}${path}`)) {
+        return actions.push({
+          type: 'remove',
+          files: 'pages/Sitemap.vue'
+        })
+      }
+    }
+
+    removePath('/pages/Sitemap.vue', actions)
+    removePath('/tide.config.js', actions)
```

Comment:
This will break the site if they have custom configs in `/tide.config.js`.

Reply:
aren't they all supposed to be moved to /tide/tide.config.js ?